src/preferences/configobject.cpp

```diff
@@ -306,6 +337,41 @@ double ConfigObject<ConfigValue>::getValue(
     return ok ? result : default_value;
 }
 
+template<>
+template<>
+mixxx::RgbColor::optional_t ConfigObject<ConfigValue>::getValue(
+        const ConfigKey& key, const mixxx::RgbColor::optional_t& default_value) const {
+    const ConfigValue value = get(key);
+    if (value.isNull()) {
+        return default_value;
+    }
+    auto color = mixxx::RgbColor::optional_t(value.value.toUInt());
```

Comment:
Use the conversion from QVariant() directly. I remember that I saw more occurrences of this pattern.

Reply:
`value.value` is not a `QVariant`. It's a `const QString&`.

Reply:
`value.value` is a `QString`, not a `QVariant`.

Reply:
How about storing hex color codes instead of integer strings, i.e. #RRGGBB?

Reply:
Done.